lib/memo_wise.rb

```diff
@@ -285,12 +301,6 @@ def memo_wise(method_name_or_hash) # rubocop:disable Metrics/PerceivedComplexity
         # hash key is just the method name.
         if method.arity.zero?
           klass.module_eval <<-END_OF_METHOD, __FILE__, __LINE__ + 1
-            # def foo
```

Comment:
Not sure why these were here but they looked like the result of testing?

Reply:
Isn't this is what Rubocop asks for demonstrating what a dynamically-evaluated method will look like?

Reply:
Haha yep I just discovered that. 👍